react/react_test.go

```diff
@@ -192,6 +197,37 @@ func TestCallbackAddRemove(t *testing.T) {
 	}
 }
 
+func TestMultipleCallbackRemoval(t *testing.T) {
+	r := New()
+	inp := r.CreateInput(1)
+	c := r.CreateCompute1(inp, func(v int) int { return v + 1 })
+
+	numCallbacks := 5
+
+	calls := make([]int, numCallbacks)
+	handles := make([]CallbackHandle, numCallbacks)
+	for i := 0; i < numCallbacks; i++ {
+		// Rebind i, otherwise all callbacks will use i = numCallbacks
```

Comment:
maybe the comment (explaining the `i := i`) isn't necessary, maybe if someone is familiar enough with go and they see `i := i` in a loop body they know what it means

Also removing that line will break the tests (they'll all use `calls[5]` which doesn't exist) so maybe that will be compelling enough to indicate that the line shouldn't be removed. Will let reviewers make the call.